packages/cli/src/daemon-config.ts

```diff
@@ -87,7 +88,17 @@ export class DaemonHTTPApiConfig {
   /**
    * Port to listen on.
    */
-  @jsonMember(Number)
+  @jsonMember(AnyT, {serializer: inPort => {
+      const validPort = Number(inPort)
+      if (inPort == undefined || inPort == null) {
```

Comment:
nit: I'm not a js expert, but I think that it might be best practice to use `===` (three equal signs) here, as `==` does type coercion before comparing (so basically `undefined == null` is `true`, I think that may be the only practical difference in this case).